examples/Example 1.yml

```diff
@@ -0,0 +1,58 @@
+# Teams Section
+Teams:
+- Name: Team 1
+  Identity: Security Group 1
+- Name: Team 2
+  Identity: Security Group 2
+
+# Patterns
+Patterns:
+- !Security # Exclamation mark is not actual yaml, but the yaml library understands it as inheritance, parent is pattern
+  Name: Default
+  Application Groups:
+  - Name: Contributor
+    Members: Security Group {0} # {0} will be replaced with a key using a string.format
+    Namespaces:
+    - Name: Project
+      Allow:
+      - Create tag definition
```

Comment:
Want to use the display names here? Or export the underlying system constants? 

`Bypass policy when completing pull requst` => `PullRequestBypassPolicy`

I'd opt for the constants.

Reply:
Well spotted, I would also prefer the constants. Less error-prone.

Reply:
As long they are readable for the user and easily to match with the UI